src/main/kotlin/com/terraformation/backend/documentproducer/db/VariableStore.kt

```diff
@@ -255,7 +298,128 @@ class VariableStore(
       }
     }
 
-    private fun fetchNumber(base: BaseVariableProperties): NumberVariable {
+    private fun fetchSection(base: BaseVariableProperties): SectionVariable {
+      val sectionRow =
+          variableSectionsDao.fetchOneByVariableId(base.id)
+              ?: throw VariableIncompleteException(base.id)
+
+      val children =
+          dslContext
+              .select(VARIABLE_SECTIONS.VARIABLE_ID)
+              .from(VARIABLE_SECTIONS)
+              .join(VARIABLE_MANIFEST_ENTRIES)
+              .on(VARIABLE_SECTIONS.VARIABLE_ID.eq(VARIABLE_MANIFEST_ENTRIES.VARIABLE_ID))
+              .where(VARIABLE_SECTIONS.PARENT_VARIABLE_ID.eq(base.id))
+              .and(VARIABLE_MANIFEST_ENTRIES.VARIABLE_MANIFEST_ID.eq(manifestId))
+              .orderBy(VARIABLE_MANIFEST_ENTRIES.POSITION)
+              .fetch(VARIABLE_SECTIONS.VARIABLE_ID.asNonNullable())
+              .map { fetchVariable(it) as SectionVariable }
+
+      val recommends =
+          with(VARIABLE_SECTION_RECOMMENDATIONS) {
+            dslContext
+                .select(RECOMMENDED_VARIABLE_ID)
+                .from(VARIABLE_SECTION_RECOMMENDATIONS)
+                .where(SECTION_VARIABLE_ID.eq(base.id))
+                .and(VARIABLE_MANIFEST_ID.eq(manifestId))
+                .fetch(RECOMMENDED_VARIABLE_ID.asNonNullable())
+          }
+
+      return SectionVariable(base, sectionRow.renderHeading!!, children, recommends)
+    }
+
+    protected fun fetchTable(base: BaseVariableProperties): TableVariable {
```

Comment:
Should this be in the other fetch class? Tables should be entirely in the all-variables sheet, I believe.

Reply:
That is correct, thanks for the catch

Reply:
Looks like I had it in both places, will remove from this spot